Coremelysis/Coremelysis/HistoryView/History Summary View/HistorySummaryViewController.swift

```diff
@@ -0,0 +1,112 @@
+        self.negativeEntriesLabel.text = viewModel.numberOfNegativeEntries
+        self.neutralEntriesLabel.text = viewModel.numberOfNeutralEntries
+    }
+
+    // - MARK: Layout
+    private func setupNumberOfEntriesLabel() {
+        numberOfEntriesLabel.text = viewModel.numberOfEntries
+        numberOfEntriesLabel.font = .preferredFont(forTextStyle: .title1)
+        numberOfEntriesLabel.numberOfLines = 0
+        numberOfEntriesLabel.textColor = .coremelysisAccent
+        numberOfEntriesLabel.textAlignment = .center
+
+        view.addSubview(numberOfEntriesLabel)
+    }
+
+    private func setupEntriesLabels() {
+        positiveEntriesLabel.textColor = .coremelysisPositive
+        negativeEntriesLabel.textColor = .coremelysisNegative
+        neutralEntriesLabel.textColor = .coremelysisNeutral
+
+        positiveEntriesLabel.font = .preferredFont(forTextStyle: .headline)
+        negativeEntriesLabel.font = .preferredFont(forTextStyle: .headline)
+        neutralEntriesLabel.font = .preferredFont(forTextStyle: .headline)
+
+        positiveEntriesLabel.text = viewModel.numberOfPositiveEntries
+        negativeEntriesLabel.text = viewModel.numberOfNegativeEntries
+        neutralEntriesLabel.text = viewModel.numberOfNeutralEntries
+
+        positiveEntriesLabel.textAlignment = .center
+        negativeEntriesLabel.textAlignment = .center
+        neutralEntriesLabel.textAlignment = .center
+
+        view.addSubview(positiveEntriesLabel)
+        view.addSubview(negativeEntriesLabel)
+        view.addSubview(neutralEntriesLabel)
+    }
+
+    private func layoutConstraints() {
+        NSLayoutConstraint.activate([
+            numberOfEntriesLabel.topAnchor.constraint(equalTo: view.topAnchor),
+            numberOfEntriesLabel.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 0.5),
+            numberOfEntriesLabel.widthAnchor.constraint(equalTo: view.widthAnchor),
+            numberOfEntriesLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+
+            positiveEntriesLabel.topAnchor.constraint(equalTo: numberOfEntriesLabel.bottomAnchor,
+                                                      constant: DesignSystem.Spacing.default),
+            positiveEntriesLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            positiveEntriesLabel.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            positiveEntriesLabel.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 0.3),
+
+            negativeEntriesLabel.topAnchor.constraint(equalTo: numberOfEntriesLabel.bottomAnchor,
+                                                      constant: DesignSystem.Spacing.default),
+            negativeEntriesLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            negativeEntriesLabel.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            negativeEntriesLabel.widthAnchor.constraint(equalTo: view.widthAnchor,
```

Comment:
Maybe you could have used a stack view or made the labels in this row constrain to the width of the first one to ensure their equal sizes.

Reply:
Good catch! I will add this change on the next PR.